Gemfile

```diff
@@ -20,7 +20,9 @@ gem 'scoped_search', '>= 4.1.8', '< 5'
 gem 'ldap_fluff', '>= 0.5.0', '< 1.0'
 gem 'apipie-rails', '>= 0.5.17', '< 0.6.0'
 gem 'apipie-dsl', '>= 2.2.6'
-gem 'rdoc'
+# Pin rdoc to prevent updating bundled psych (https://github.com/ruby/rdoc/commit/ebe185c8775b2afe844eb3da6fa78adaa79e29a4)
+# Pin psych is needed for Rails 6.0, Rails 6.1 should work with Psych > 4
```

Comment:
Technically, we're not pinning Psych (there is no psych in this PR and none is listed in Gemfile.lock once you've run bundler), but only ensuring we're not pulling in rdoc that needs psych at all.

But this is splitting hairs and I don't mind either wording